src/languages/markup.ts

```diff
@@ -103,7 +103,7 @@ export default {
 					'style': inlineEmbedded('style', 'css'),
 					'script': inlineEmbedded('script', 'javascript'),
 				},
-				'tag/attr-value': {
+				'tag/inside/attr-value': {
```

Comment:
It was `tag.inside` in the original grammar.

Reply:
The idea is that `inside` will be applied automatically (since there's no point in manipulating that object)

Reply:
```suggestion
				'tag/attr-value': {
```

Reply:
> The idea is that `inside` will be applied automatically (since there's no point in manipulating that object)

@LeaVerou, is this something you are planning to implement? Because without the fix, we get this error when trying to highlight a markup code:

<img width="474" alt="image" src="https://github.com/user-attachments/assets/73ab2fa2-521d-45e5-a58d-fcaaeec81a38" />

And none of the code blocks that follow the markup code are highlighted.

Reply:
Yes

Reply:
Or you can :)